scipy/interpolate/_bary_rational.py

```diff
@@ -597,8 +597,7 @@ def clean_up(self, cleanup_tol=1e-13):
 
 
 class FloaterHormannInterpolator(_BarycentricRational):
-    r"""
-    Floater-Hormann barycentric rational interpolation.
+    r"""Floater-Hormann barycentric rational interpolator (C∞ smooth).
```

Comment:
I don't know how precise you want to be but it's only smooth on the real line, in the complex plane it's meromorohic 
```suggestion
    r"""Floater-Hormann barycentric rational interpolator (C∞ smooth for real data).
```